src/posts/keymaps/dchenex.md

```diff
@@ -0,0 +1,24 @@
+---
+author: dchenex
+baseLayouts: QWERTY, Gallium
+firmwares: ZMK
+hasHomeRowMods: false
+hasLetterOnThumb: false
+hasRotaryEncoder: true
+isAutoShiftEnabled: false
+isComboEnabled: true
+isSplit: true
+isTapDanceEnabled: false
+keybindings: Graphics/CAD
+keyboard: Kyria
+keyCount: 50
+keymapImage: https://github.com/user-attachments/assets/c328c739-d2cb-4cf2-a8df-c2e2a926c70f
```

Comment:
The `keymapImage` should be a direct link (i.e. a URL ending on a file extension like .png or .jpg and without any UI around). I would have made the necessary conversion myself if it weren't for the fact that your provided link gives a 404 Not Found error message.